common/utils/src/main/resources/error/error-classes.json

```diff
@@ -3854,6 +3854,11 @@
           "Catalog <catalogName> does not support <operation>."
         ]
       },
+      "COLLATION" : {
+        "message" : [
+          "Collation support is under development and not all features are supported. To enable existing features set <collationEnabled> to `true`."
```

Comment:
when we turn off this feature, it means we do not want to expose it to the end users and it's weird if we still tell them how to enable it.

How about just say `String collation is not supported yet.`?

Reply:
+1 on what @cloud-fan suggest. Let's make it simple and just mention that it is unsupported.